src/views/studio/StudioLayout.tsx

```diff
@@ -147,7 +147,9 @@ const StudioLayout = () => {
   )
 }
 
-const MainContainer = styled.main`
+const MainContainer = styled.main<{ hasSidebar: boolean }>`
+  --size-sidenav-width-collapsed: ${({ hasSidebar }) => (hasSidebar ? '72px' : 0)};
```

Comment:
This will break the mobile views because you set this to constant `72px` although it changes to 0 on mobile. We should use the original `--size-sidenav-width-collapsed` value here that changes automatically. However that means we need to introduce a separate variable